server/src/controllers/poaching-incident.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { PoachingIncidentService } from '../services/poaching-incident.service';
+
+@Controller()
+export class PoachingIncidentController {
+  constructor(
+    private readonly poachingIncidentService: PoachingIncidentService,
+  ) {}
+
+  @Get('addPoachingIncident')
+  async addPoachingIncident(
```

Comment:
Async not needed here - or you can add "return await" to line 16

Reply:
Done